README.md

```diff
@@ -98,5 +105,67 @@ Insert the `Save Build Scan` step after each Maven execution step in the Github
       - name: Build with Maven
         run: mvn clean package
       - name: Save Build Scan
-        uses: gradle/github-actions/maven/build-scan-save@v1.0
-[...]```
+        uses: gradle/github-actions/maven/build-scan/save@v1.0
+[...]
+```
+
+## maven/build-scan/publish
+
+A composite action to publish all Maven Build Scans® saved as workflow artifacts when validating a pull-request (by the `maven/build-scan/save` action). 
```

Comment:
Better:
"This action will publish all Maven Build Scans® that have been saved as workflow artifacts by the `maven/build-scan/save` action."